config.example.yaml

```diff
@@ -119,7 +120,22 @@ templates:
                   get:
                     backend_request:
                       uri: http://graphoid.wikimedia.org/{domain}/v1/png/{title}/{revision}/{graph_id}
-                      body: '{$.request.body}'
+
+      /{module:mathoid}:
+        x-modules:
+          - name: simple_service
+            version: 1.0.0
+            type: file
+            options:
+              paths:
+                /v1/{format}/{formula}/{type}:
+                  get:
+                    backend_request:
+                      method: post
+                      uri: http://mathoid-tester.wmflabs.org/{format}
```

Comment:
Ok that's good to know. I normally regulary delete labs-instances and create new ones.
Maybe we can use http://mathoid.testme.wmflabs.org which is the URL currently used by the math extension
See also
https://phabricator.wikimedia.org/T87390